### PR TITLE
Prepare for v0.9.5

### PR DESCRIPTION
**Pull request by davidatwhiletrue**
### Summary
- Code documentation and documentation site configs.
- Integration and unit tests.
- Bugfix: deserialization of a deploy with StoredVersionedContractBy* executable deploy item failed because of a typo.
- Serialize BigIntegers as unsigned always.

### Checklist

- [X] Code is properly formatted
- [X] All commits are signed
- [X] Tests included/updated or not needed
- [X] Documentation (manuals or wiki) has been updated or is not required

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -15,25 +37,26 @@ dotnet build --configuration Release
 To run the tests, use this command:
 
 ```
-dotnet test --settings Casper.Network.SDK.Test/test.runsettings
+dotnet test --filter "TestCategory!~NCTL"
```
Comment (ihor) [inline README.md:40]: @davidatwhiletrue the command above fails becuse of double quotes on Windows:
```
$ dotnet test --filter "TestCategory!~NCTL"
zsh: event not found: ~NCTL
```

Can we changes it to single quotes? Do they work on Windows?

**Review by ihor — CHANGES_REQUESTED** [review]

**Comment by ihor** [comment]
@davidatwhiletrue another thing I noticed is that we should add a link to the dockerized NCTL instructions in the README to make it easier to set up the test environment. Atm one should know how to use NCTL in order to proceed. Do you agree? Please make a note to add it once it's ready if you do.

File: README.md (hunk 1 above)
Comment (davidatwhiletrue) [reply README.md]: Changed to single quotes.

File: README.md [hunk 2]
```diff
@@ -15,25 +37,26 @@ dotnet build --configuration Release
...
 
-## Usage Examples (WIP)
+3. Run the tests:
 
-* [Counter contract tutorial with C#](./Docs/Tutorials/counter-contract/README.md)
-* [Key-Value storage tutorial with C#](./Docs/Tutorials/kvstorage-contract/README.md)
-* [Key management](./Docs/Articles/KeyManagement.md )
-* [Working with CLValue](./Docs/Articles/WorkingWithCLValue.md)
+```
+dotnet test --settings Casper.Network.SDK.Test/test.runsettings --filter "TestCategory~NCTL"
```
[the remark this answers is not in the document]
Comment (davidatwhiletrue) [reply README.md]: Changed to single quotes.

**Comment by davidatwhiletrue** [comment]
@ihor It's a good idea. I've added to the README the command to start the container and a link to the full instructions. Similarly, I added a new page to the documentation website.

**Review by ihor — APPROVED** [review]

**Pull request merged**